src/tiny_slam/tiny_world.h

```diff
@@ -30,13 +40,25 @@ class TinyWorld : public LaserScanGridWorld {
   using Point = DiscretePoint2D;
 public:
 
+  /*!
+   * Initialize the world to produce tiny SLAM.
+   * \param[in] gcs    - shared pointer to a strategy for the each cell.
+   *                   (how does estimate the occupancy, cost of laser data etc)
+   * \param[in] params - the initial values of trust for laser data.
```

Comment:
The description is based on current TinyWorldParams's members. It should be more general.